SpectrumVisualization.py

```diff
@@ -0,0 +1,15 @@
+import numpy
+from matplotlib.pyplot import semilogy
+from matplotlib import pyplot as plot
+
```

Comment:
Is this file redundant with `Spectrum_Peak_Visualization.py`? If so, remove it; if not, combine it so there is just one file.